packages/net-stubbing/lib/server/util.ts

```diff
@@ -18,12 +19,43 @@ import type { CypressIncomingRequest } from '@packages/proxy'
 import type { InterceptedRequest } from './intercepted-request'
 import { caseInsensitiveGet, caseInsensitiveHas } from '../util'
 
-// TODO: move this into net-stubbing once cy.route is removed
-import { parseContentType } from '@packages/server/lib/controllers/xhrs'
 import type { CyHttpMessages } from '../external-types'
 import { getEncoding } from 'istextorbinary'
 
 const debug = Debug('cypress:net-stubbing:server:util')
+const htmlLikeRe = /<.+>[\s\S]+<\/.+>/
+
+const isValidJSON = function (text) {
+  if (_.isObject(text)) {
+    return true
+  }
+
+  try {
+    const o = JSON.parse(text)
+
+    return _.isObject(o)
+  } catch (error) {
+    false
+  }
+
+  return false
+}
+
+export function parseContentType (response?: string) {
+  const ret = (type) => {
```

Comment:
Do we really get anything out of having this be its own function?
```suggestion
  const ret = (type: string) => {
```